giraffe/src/utils/newTable.ts

```diff
@@ -100,3 +101,16 @@ class SimpleTable implements Table {
 }
 
 export const newTable = (length: number): Table => new SimpleTable(length)
+
+export const newTableFromConfig = (config: Config): Table => {
+  if (!config) {
+    return newTable(0)
+  }
+  if (config.table) {
+    return config.table
+  }
+  if (config.fluxResponse) {
+    return fromFlux(config.fluxResponse).table
```

Comment:
i'm digging this logic, it's super clean!

when would `newTable` be called with a `config` that has a `table`? Could that be an indicator of repeated function calls?

Reply:
Normally, it `newTable` would not be called with a `config` that already has a `table`. This is to protect against edge cases and misuse.